src/paho/mqtt/client.py

```diff
@@ -131,25 +161,49 @@
 mqtt_ms_send_pubrec = 8
 mqtt_ms_queued = 9
 
+
 # Error values
-MQTT_ERR_AGAIN = -1
-MQTT_ERR_SUCCESS = 0
-MQTT_ERR_NOMEM = 1
-MQTT_ERR_PROTOCOL = 2
-MQTT_ERR_INVAL = 3
-MQTT_ERR_NO_CONN = 4
-MQTT_ERR_CONN_REFUSED = 5
-MQTT_ERR_NOT_FOUND = 6
-MQTT_ERR_CONN_LOST = 7
-MQTT_ERR_TLS = 8
-MQTT_ERR_PAYLOAD_SIZE = 9
-MQTT_ERR_NOT_SUPPORTED = 10
-MQTT_ERR_AUTH = 11
-MQTT_ERR_ACL_DENIED = 12
-MQTT_ERR_UNKNOWN = 13
-MQTT_ERR_ERRNO = 14
-MQTT_ERR_QUEUE_SIZE = 15
-MQTT_ERR_KEEPALIVE = 16
+class MQTTErrorCode(enum.IntEnum):
```

Comment:
Maybe this should be in a new module, say, `mqtt.enums`? The integer aliases could remain here for compatibility.